js/ui/status/keyboard.js

```diff
@@ -480,6 +481,41 @@ const InputSourceManager = new Lang.Class({
             this._updateMruSettings();
     },
 
+    passwordMode: function(enable) {
```

Comment:
You could probably make this a property instead of a method

Reply:
To me this is not a property. This is switching the keyboard layout to a latin one and keeps track of the current layout to be able to switch back. Maybe the method should be named differently? "switchKeyboardLayout(['default'/'password'])"

Reply:
I'd be OK to rename the method to `setPasswordMode()`; but that would be equivalent to having a `passwordMode` property and setting it to `true`. `setPasswordLayout()` could also work.